app/assets/stylesheets/views/_brexit_checker_account_signup.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-experiment > .govuk-inset-text {
+  border-color: #1d70b8;
+}
+
+#account-signup {
```

Comment:
Styles shouldn't be applied using element IDs. Can this be changed to a class instead?

Reply:
amended, now uses a class